report_backend/base.py

```diff
@@ -12,15 +12,15 @@ def __init__(self, **kwargs):
 
 
     @abc.abstractmethod
-    def json_report_to_csv(self, json_report_data={}):
+    def json_report_to_csv(self, json_report_data):
         """
         Process json data to convert into csv format.
         """
         raise NotImplementedError()
 
 
     @abc.abstractmethod
-    def create_csv_file(self, course, body_dict):
+    def create_csv_file(self, file_name, body_dict, headers):
```

Comment:
No, since it's a change in the base class and the other classes are overwriting his base behaviour and i've tested in local with the other reports.
But the other ones needs this refactoring in order to save correctly in S3 storage. @andrey-canon